app/Repositories/Team/TeamRepositoryInterface.php

```diff
@@ -60,6 +70,16 @@ public function removeTeamProjectUser($team, $user);
      */
     public function removeTeamUserProject($team, $project);
 
+    /**
+     * Remove invited user
+     *
+     * @param $team
+     * @param $email
+     */
+    public function removeInvitedUser($team, $email);
+
+
+
```

Comment:
Remove additional blank lines.

Reply:
Removed the additional blank lines at TeamRepositoryInterface@removeInvitedUser().